src/dispatch/incident/models.py

```diff
@@ -133,9 +136,22 @@ def incident_faq(self):
                     return d
 
     @hybrid_property
-    def last_status_report(self):
-        if self.status_reports:
-            return sorted(self.status_reports, key=lambda r: r.created_at)[-1]
+    def last_tactical_report(self):
+        if self.reports:
```

Comment:
Do we need a property to fetch all tactical reports (not just last)? Perhaps we break this function into two hybrid properties that call each other (tactical_reports and last_tactical_reports).

Reply:
Sure, I can break them into two.